locations/spiders/talbots.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class TemplateSpider(scrapy.Spider):
+    name = "talbots"
+    allowed_domains = ["www.talbots.com"]
+    start_urls = [
+        "https://www.talbots.com/view-all-stores/"
+    ]
+
+    def parse(self, response):
+        all_data = response.xpath('//*[contains(@class, "store-row")]').getall()
+        for data in all_data:
+            resp = scrapy.Selector(text=data)
+            properties = {
+                "name": self.sanitize_name(resp.xpath('//a[contains(@class, "store-details-link")]//text()').get()),
+                "phone": resp.xpath('//a[contains(@href, "tel")]/text()').get(),
+                "addr_full": self.get_address(resp.xpath('//div[contains(@class, "store-name")]/following-sibling::text()').getall()),
+                "opening_hours": self.sanitize_time(resp.xpath('//*[contains(@class, "store-hours storeCol")]//text()').getall()),
```

Comment:
Ideally this would be in the OpenStreetMap format, we have an `OpeingHours` class that can help format it.

Reply:
`OpeningHours` class expects `open` and `close` time to be in 24-hr format but this site gives time information in [12hrs format](https://www.talbots.com/view-all-stores/)
```
HOURS:
Mon 10:00 AM - 7:00 PM
Tues 10:00 AM - 7:00 PM
Wed 10:00 AM - 7:00 PM
Thurs 10:00 AM - 7:00 PM
Fri 10:00 AM - 7:00 PM
Sat 10:00 AM - 7:00 PM
Sun 12:00 PM - 6:00 PM
```
I can convert it to 24hr format before but it will complicate the code. Let me know if it is needed.